packages/features/calendars/lib/getTodaysDateInTimeZone.ts

```diff
@@ -0,0 +1,27 @@
+import timezone from "dayjs/plugin/timezone";
+import utc from "dayjs/plugin/utc";
+
+import dayjs from "@calcom/dayjs";
+
+// with new Date(), we get browser's current date.
+// The below function gets today's date w.r.t to selected timezone.
+// Example :
+// Consider browser current date-time is - "Sat Jul 20 2024 01:00:00 AM (Asia/Kolkata GMT +5:30)".
+// And, if preferred timezone is "Pacific/Pago GMT -11:00", we need to get today's date as "Fri Jul 19".
```

Comment:
same

Reply:
reduced to 1-2 lines, as i feel it's important and also for someone who is sees/reviews this code for first time. 
The question may come for anybody seeing this for first time, why not use already existing function to get current date in timezone. 
That's why had added detailed explanation, now reduced to just 2 lines.